Update AddPupilForm

PReichetanz opened the pull request:
- Rename Form to AddPupilForm
-  Delete category and evaluation input
- Update types and type references
- Update addPupil function in usePupils

PReichetanz committed:
Update AddPupilForm

- Rename Form to AddPupilForm
-  Delete category and evaluation input
- Update types and type references
- Update addPupil function in usePupils

F-Kirchhoff commented on src/app/components/AddPupilForm/AddPupilForm.tsx:
```diff
@@ -3,39 +3,25 @@ import styled from 'styled-components';
-    category: string;
-    evaluation: string;
-  }) => void;
+  onSubmit: (pupil: { name: string }) => void;
```
maybe you can omit the object and set pupil to a string?

F-Kirchhoff reviewed (approved):
nice :3

PReichetanz committed:
Update according to review

- AddPupilForm now submits a string --> update all references
- Delete RawInput type
